entrypoint.sh

```diff
@@ -51,6 +56,24 @@ case ${RUNNER_SCOPE} in
     ;;
 esac
 
+# If the variable is not set, set it with the default value
+if [ -z "${CONFIGURED_ACTIONS_RUNNER_FILES_DIR}" ]; then
+  CONFIGURED_ACTIONS_RUNNER_FILES_DIR="/actions-runner-files"
+fi
+
+# Loading the files from the mounted directory
+if [ -d "${CONFIGURED_ACTIONS_RUNNER_FILES_DIR}" ]; then
+  cp -p -r "${CONFIGURED_ACTIONS_RUNNER_FILES_DIR}/." "/actions-runner"
+fi
+
+if [ -f "/actions-runner/.runner" ]; then
+  echo "The runner has already been configured"
+  unset ACCESS_TOKEN
+  unset RUNNER_TOKEN
+  execute_docker_command $@ 
```

Comment:
im not sure what `execute_docker_command` is

Reply:
See the `run_runner` thread